core/templates/components/skill-selector/merge-skill-modal.controller.spec.ts

```diff
@@ -41,14 +41,15 @@ describe('Merge Skill Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
+  it('should initialize correctly $scope properties after controller' +
+    ' initialization', function() {
     expect($scope.skillSummaries).toBe(skillSummaries);
     expect($scope.categorizedSkills).toBe(categorizedSkills);
     expect($scope.allowSkillsFromOtherTopics).toBe(true);
     expect($scope.selectedSkillId).toBe('');
   });
 
-  it('should close modal on saving', function() {
+  it('should close modal on save', function() {
     $scope.save();
     expect($uibModalInstance.close).toHaveBeenCalledWith({
       skill: skill,
```

Comment:
Unable to comment below, but line 60, make it should close modal when confirm button is clicked

Reply:
I've removed this spec because `save` calls `confirm`, so it's not necessary to test `confirm` behavior.